src/Avatar/Avatar.js

```diff
@@ -22,6 +22,7 @@ function getStyles(props, context) {
       borderRadius: '50%',
       height: size,
       width: size,
+      boxShadow: src ? `inset 0px 0px 0px 1px ${avatar.borderColor}` : 'none',
```

Comment:
yes, the visual impact. I can't see any difference with or without the property on Chome v50 OSX.